src/Control/HTTPResponse.php

```diff
@@ -109,12 +114,31 @@ class HTTPResponse
      * @param string $statusDescription The text to be given alongside the status code.
      *  See {@link setStatusCode()} for more information.
      */
-    public function __construct($body = null, $statusCode = null, $statusDescription = null)
+    public function __construct($body = null, $statusCode = null, $statusDescription = null, $version = null)
     {
         $this->setBody($body);
         if ($statusCode) {
             $this->setStatusCode($statusCode, $statusDescription);
         }
+        if (!$version) {
```

Comment:
Was this handled by `HTTP` before, or just an unrelated improvement? Can't see mentions of `SERVER_PROTOCOL` elsewhere in the diff

Reply:
I don't think we need it, but it's PSR-7 so I'm inclined to let @dhensby's enhancement sneak in.